src/main/java/com/example/onlinebookstore/model/User.java

```diff
@@ -0,0 +1,74 @@
+package com.example.onlinebookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
```

Comment:
```suggestion
@Entity
@Getter
@Setter
```

Do we need soft delete approach here?